tyk-headless/tests/deployment-gw-repset_test.yaml

```diff
@@ -0,0 +1,294 @@
+suite: test headless gateway deployment
+templates:
+  - templates/configmap-gateway.yaml
+  - templates/deployment-gw-repset.yaml
+  - templates/secrets.yaml
+  - templates/secret-certs.yaml
+release:
+  name: my-release
+  namespace: my-namespace
+chart:
+  version: 0.12.0
```

Comment:
Do we have to specify version? It doesn't run on the charts on the latest commit

Reply:
you are right. we do not need to specify a chart version. I will remove it 👍